### DISCUSSION TITLE
Copy sync_dir to new workstation

### DISCUSSION THREAD

**Discussion by JessiBaughman** [abraunegg/onedrive#1918] [Q&A] [answered]
Can I tar (to preserve permissions) a sync_dir from one workstation and put it on another to speed up the first sync to Sharepoint? Or since my sync_dir is on its own drive on each workstation, I could clone the drives instead. Would the first sync on the 2nd workstation recognize the local files and just update its database or does it need to actually download from Sharepoint to populate the database correctly? If it matters, I have it configured as a download-only sync.

My sync_dir is ~80GB and I want to set this up on two other workstations, so I'm just looking to see if I can speed things up a bit.

**Comment by abraunegg** [comment, marked as answer]
@JessiBaughman 

> Can I tar (to preserve permissions) a sync_dir from one workstation and put it on another to speed up the first sync to Sharepoint? Or since my sync_dir is on its own drive on each workstation, I could clone the drives instead. Would the first sync on the 2nd workstation recognize the local files and just update its database or does it need to actually download from Sharepoint to populate the database correctly? If it matters, I have it configured as a download-only sync.
 
If all the workstations are on your local network, you can use 'rsync' locally to sync all your local 'sync_dir' on all your workstations.

If a file exists locally, and the size + hash matches, the file is not downloaded, and the application will just populate the sync database.

**Reply by JessiBaughman** [reply]
Thanks! Just wanted to verify before doing the first sync.

**Reply by abraunegg** [reply]
@JessiBaughman 
You can also copy the 'items.sqlite3' file from the `~/.config/onedrive` location on the source PC where all the downloads are done, to the other workstations as well - there is nothing special in that file that locks it to a specific machine, and would also save time processing data online - if there are no differences in the SharePoint data.
